portfolio/src/main/java/com/google/sps/utils/CommentDataStore.java

```diff
@@ -0,0 +1,37 @@
+package com.google.sps.utils;
+
+import com.google.appengine.api.datastore.*;
+import com.google.sps.data.Comment;
+import com.google.sps.data.EntityConvertible;
+import com.google.sps.data.EntityConvertibleCreator;
+
+/** Provides the service to interact with Datastore to store or load comment. */
+public class CommentDataStore {
```

Comment:
Make it final. And maybe abstract one step further and define a generic object store interface? Then comment data store can implement that.
e.g
public final class CommentDataStore implements ObjectDataStore<Comment>

Reply:
Fixed